projects/core/src/occ/adapters/site-context/default-occ-site-context-config.ts

```diff
@@ -11,6 +15,8 @@ export const defaultOccSiteContextConfig: OccConfig = {
           'countries/${isoCode}/regions?fields=regions(name,isocode,isocodeShort)',
         baseSites:
           'basesites?fields=DEFAULT,baseSites(defaultPreviewCatalogId,defaultPreviewCategoryCode,defaultPreviewProductCode,urlEncodingAttributes)',
+        baseSitesForConfig:
+          'basesites?fields=baseSites(uid,defaultLanguage(isocode),urlEncodingAttributes,urlPatterns,stores(currencies(isocode),defaultCurrency(isocode),languages(isocode),defaultLanguage(isocode)),theme,defaultPreviewCatalogId,defaultPreviewCategoryCode,defaultPreviewProductCode)',
```

Comment:
It looks like smart edit should work fine even when the preview codes are loaded in advance in the initial multi-site call.

But would you mind just double checking, if smart edit preview pages work properly both:
- when the auto multi-site config is enabled
- when static base site config is applied in Spartacus
?

Reply:
I did the test, smartedit works fine.